app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Api
+  module V1
+    class RentsController < ApplicationController
+      before_action :authenticate_api_v1_user!
+
+      def index
+        @rents = Rent.all
+        render_paginated @rents, each_serializer: RentSerializer
```

Comment:
`render_paginated Rent.all` uses RentSerializer by default, but check on apiary if you don't need a RentIndexSerializer